### PR TITLE
Update rules for token bariers

### PR DESCRIPTION
**Pull request by babenek**
## Description

Please include a summary of the change and which is fixed.

- Updated bariers for tokens. Often ``/`` is used right with the token in url e.g. /AKIA...
- do not skip *.rst files in scan - they are text
- applied multiprocessing download for benchmark

## How has this been tested?

Please describe the tests that you ran to verify your changes.

- [x] UnitTest
- [x] Benchmark

### CODE REVIEW

File: .github/workflows/benchmark.yml [hunk 1]
```diff
@@ -22,7 +22,8 @@ jobs:
     - name: Checkout CredData
       uses: actions/checkout@v3
       with:
-        repository: Samsung/CredData
+        repository: babenek/CredData
+        ref: tokenbar
```
Comment (babenek) [inline .github/workflows/benchmark.yml:26]:
```suggestion
        repository: Samsung/CredData
```

File: .github/workflows/benchmark.yml [hunk 2]
```diff
@@ -62,7 +63,8 @@ jobs:
...
```
Comment (babenek) [inline .github/workflows/benchmark.yml:67]:
```suggestion
        repository: Samsung/CredData
```

File: .github/workflows/benchmark.yml [hunk 3]
```diff
@@ -166,7 +168,8 @@ jobs:
...
```
Comment (babenek) [inline .github/workflows/benchmark.yml:172]:
```suggestion
        repository: Samsung/CredData
```

**Review by babenek — COMMENTED** [review]
TODO: rollback after https://github.com/Samsung/CredData/pull/66

**Review by xDizzix — APPROVED** [review]
+

**Review by iuriimet — APPROVED** [review]

**Pull request merged**